engine/api/metrics/metrics.go

```diff
@@ -88,11 +96,41 @@ func Initialize(c context.Context, DBFunc func() *gorp.DbMap, instance string) {
 				countGauge(DBFunc(), *queue, "waiting", "50_more_2min_less_5min", query, now5min, now2min)
 				countGauge(DBFunc(), *queue, "waiting", "60_more_5min_less_10min", query, now10min, now5min)
 				countGauge(DBFunc(), *queue, "waiting", "70_more_10min", queryOld, now10min)
+
+				processStatusMetrics(c, DBFunc, *status)
 			}
 		}
 	}(c, DBFunc)
 }
 
+func processStatusMetrics(c context.Context, DBFunc func() *gorp.DbMap, v prometheus.GaugeVec) {
+	srvs, err := services.All(DBFunc())
+	if err != nil {
+		log.Error("Error while getting services list: %v", err)
+		return
+	}
+	mStatus := ComputeGlobalStatus(srvs)
+	apis := make(map[string]int)
+	for _, line := range mStatus.Lines {
```

Comment:
i don't understand what is going on...

Reply:
// rename api_foobar to api_0, api_1, etc...
// this will avoid creating series with custom name